ios/rüm/LastTaskCell.swift

```diff
@@ -60,7 +71,17 @@ class LastTaskCell: UITableViewCell {
             if let data = NSData(contentsOfURL: imageUrl) {
                 let image = UIImage(data: data)
                 self.kudosButton.image = image
-                self.backgroundImageView.image = image
+                dispatch_async(dispatch_get_global_queue(QOS_CLASS_USER_INITIATED, 0)) {
```

Comment:
I don't know too much about multithreading in Swift, but should this be two separate calls so we can get the photos in parallel? Like

```Swift
dispatch_async( ... get profile photo ... )
dispatch_async( ...get background photo ... )
```